main.js

```diff
@@ -0,0 +1,18 @@
+const navbarHamburger = document.querySelector('#navbar-hamburger');
+const mobileMenu = document.querySelector('#mobile-menu');
+const cross = document.querySelector('#cross');
+navbarHamburger.addEventListener('click', () => {
```

Comment:
- [ ] Also you hide the mobile menu when the website load so the user doesn't have to close it at first.

```javascript
mobileMenu.style.display = 'none';
```